contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -197,4 +201,13 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
 
     return blockedSenders;
   }
+
+  function updateFromPrevious() external override {
```

Comment:
This should definitely not be callable by anyone. Add an owner only check.

Reply:
oh yeah that would be bad. Thank you.